g2p/cli.py

```diff
@@ -766,18 +769,23 @@ def show_mappings(lang1, lang2, verbose, csv):
 
     else:
         mappings = (
-            Mapping(in_lang=m["in_lang"], out_lang=m["out_lang"])
+            Mapping.find_mapping(in_lang=m.in_lang, out_lang=m.out_lang)
             for m in MAPPINGS_AVAILABLE
         )
 
     file_type = "csv" if csv else "json"
     if verbose:
         for m in mappings:
-            json.dump(m.kwargs, sys.stdout, indent=4, ensure_ascii=False)
+            json.dump(
+                json.loads(m.model_dump_json()),
+                sys.stdout,
+                indent=4,
+                ensure_ascii=False,
+            )
```

Comment:
correct - we can't format it properly otherwise. I agree that it's silly, but this doesn't happen very often and it doesn't actually result in a performance problem

Reply:
Note that `ensure_ascii=False` is the default for `model_dump_json` ... I went through this problem a while back: https://github.com/pydantic/pydantic/issues/6800

Reply:
oh interesting - does indent work too? in that case I guess we can simplify this

Reply:
The documentation says it accepts an indent parameter, yes, so it should. That would make the code nicer.

Reply:
Just tested, and this is actually not quite right anyway: since the rules are now loaded in the mapping, we should no longer print them a second time.
And we should print them with our formatter enabled.